src/plans/quasi_newton_plan.jl

```diff
@@ -467,21 +465,50 @@ function QuasiNewtonLimitedMemoryDirectionUpdate(
         convert(mT, scale),
         project,
         vector_transport_method,
+        "",
     )
 end
+get_message(d::QuasiNewtonLimitedMemoryDirectionUpdate) = d.message
+function status_summary(d::QuasiNewtonLimitedMemoryDirectionUpdate{T}) where {T}
+    s = "limited memory $T (size $(length(d.memory_s)))"
+    (d.scale != 1.0) && (s = "$(s) initial scaling $(d.scale)")
+    d.project && (s = "$(s), projections, ")
+    s = "$(s)and $(d.vector_transport_method) as vector transport."
+    return s
+end
 function (d::QuasiNewtonLimitedMemoryDirectionUpdate{InverseBFGS})(mp, st)
+    (length(d.message) > 0) && (d.message = "") # reset message
     M = get_manifold(mp)
     p = get_iterate(st)
     r = copy(M, p, get_gradient(st))
     m = length(d.memory_s)
     m == 0 && return -r
     for i in m:-1:1
-        d.ρ[i] = 1 / inner(M, p, d.memory_s[i], d.memory_y[i]) # 1 sk 2 yk
+        # what if we dvide by zero here? Setting to zero ignores this in the next step
+        #precompute in case inner is expensive
+        v = inner(M, p, d.memory_s[i], d.memory_y[i]) # 1 sk 2 yk
+        if v ≈ 0
```

Comment:
but then I would use `== 0`?

Reply:
...besides that I was hoping to just maybe finish it today, but now `Circle` is annoying me in about a trillion cases since I assume the max-stepize to be a float but circles infectivity radius is an irrational 🙄

Reply:
Sure, `==0` is also effectively the same.

BTW, `pi` should be converted to a float very easily, it shouldn't cause that many issues?

Reply:
Well, took me a bit to fix the pi.

And I switched to iszero, so I'd call this PR ready :)